large-borrows-governance/src/autotask.js

```diff
@@ -0,0 +1,195 @@
+// LARGE BORROWS GOVERNANCE AGENT ALERT - note there's never been an alert to this agent yet
+// agent id - 0x704a35d97dfae2caf3b409206f1bc4c8a06671170108d07f9b3c7ea5026916fc
+
+const axios = require('axios');
+
+const fortaApiEndpoint = 'https://api.forta.network/graphql';
+const needle = require('needle')
+
+function createDiscordMessage(borrowerAddress, governanceEvent, transactionHash) {
+  const borrowerFormatted = borrowerAddress.slice(0, 6);
+
+  // // construct the Etherscan transaction link
+  const etherscanLink = `[TX](<https://etherscan.io/tx/${transactionHash}>)`;
+
+  return `${etherscanLink} 💸 **${borrowerFormatted}** has enough **COMP** tokens to pass min threshold for the governance event: **${governanceEvent}**`;
+}
+
+// post to discord
+async function postToDiscord(url, message) {
+  const method = 'post';
+  const headers = {
+    'Content-Type': 'application/json',
+  };
+  const data = JSON.stringify({ content: message });
+
+  let response;
+  try {
+    // perform the POST request
+    response = needle.post(url, {content: message}, {json: true} )
+  } catch (error) {
+    // is this a "too many requests" error (HTTP status 429)
+    if (error.response && error.response.status === 429) {
+      // the request was made and a response was received
+      // try again after waiting 5 seconds
+      // eslint-disable-next-line no-promise-executor-return
+      const promise = new Promise((resolve) => setTimeout(resolve, 5000));
+      await promise;
+      response = needle.post(url, {content: message}, {json: true} )
+    } else {
+      // re-throw the error if it's not from a 429 status
+      throw error;
+    }
+  }
+
+  return response;
+}
+
+async function getFortaAlerts(agentId, transactionHash) {
+  const headers = {
+    'content-type': 'application/json',
+  };
+
+  // construct query with data that you want to get back
+  const graphqlQuery = {
+    operationName: 'recentAlerts',
+    query: `query recentAlerts($input: AlertsInput) {
+      alerts(input: $input) {
+        pageInfo {
+          hasNextPage
+          endCursor {
+            alertId
+            blockNumber
+          }
+        }
+        alerts {
+          createdAt
+          name
+          protocol
+          findingType
+          hash
+          source {
+            transactionHash
+            block {
+              number
+              chainId
+            }
+            agent {
+              id
+            }
+          }
+          severity
+		  metadata
+		  description
```

Comment:
```suggestion
          metadata
          description
```
Replace tabs with spaces